src/stream/src/executor/aggregation/distinct.rs

```diff
@@ -180,18 +186,14 @@ impl<S: StateStore> ColumnDeduplicater<S> {
     }
 
     /// Flush the deduplication table.
-    fn flush(&mut self, dedup_table: &StateTable<S>, ctx: ActorContextRef) {
+    fn flush(&mut self, _dedup_table: &StateTable<S>) {
```

Comment:
It seems that we won't flush the `dedup_table` anymore here. May remove the parameter and also update the method comment and the method name to avoid confusion.